rts/lra/lra-coordinator-jar/src/test/java/io/narayana/lra/coordinator/LRACoordinatorRecovery1TestCase.java

```diff
@@ -68,7 +70,9 @@
 public class LRACoordinatorRecovery1TestCase extends TestBase {
     private static Package[] coordinatorPackages = {
             RecoveryModule.class.getPackage(),
+            ToggleSecurityFilter.class.getPackage(),
             Coordinator.class.getPackage(),
+            JaxRsActivator.class.getPackage(),
```

Comment:
This is the change included for JWT changes, otherwise there are test failure. `JaxRsActivator` has to be included in this test class

Reply:
This answers my doubt a bit but not truly. There had to be some functionality making test to start the JAX-RS application. I mean there could be two such configuration, duplicating each other.
What was the JAX-RS `javax.ws.rs.core.Application` configuration for this test before?

Reply:
For introducing security layer, I have added `@LoginConfig(authMethod = "MP-JWT", realmName = "narayana-lra-jwt")` in `JaxRsActivator` class, this is causing the test failure.
Therefore, we need to include this.

Reply:
ok,  would like to understand why it was working before and how the JAX-RS app was activated when there was not added the `JaxRSActivator`. The `JaxRSActivator` was not deployed - the `Application` of the activator was not part of the deployment, despte there was not the `@LoginConfig` still there was not added the `Application` from the activator class.
Are there two `Application` classes in the deployment now?

Reply:
There is still single Application class in the deployment.
When test calls these secured endpoints, it would like to know, what kind of security are we going to use, as we have used `@RolesAllowed` on all the endpoints. So after this it will be pointed towards `JaxRSActivator` class `@LoginConfig` annotation, and at this point we have defined `authMethod = "MP-JWT"` .
I think when the secured endpoints are called, it is not able to get to the type of `authMethod` untill we add `JaxRSActivator`. This is just an observation, but I could be wrong.

Reply:
Ok, I've got to this to check how is it and you are right there are no two activators as the activator is already added by package `Coordinator.class.getPackage()`. Please remove the line
```
JaxRsActivator.class.getPackage()
```
as it's not needed and the test passes without it as well.